FoodTruckFinder/src/main/java/org/abaez/repositories/FoodTruckRepository.java

```diff
@@ -0,0 +1,21 @@
+package org.abaez.repositories;
+
+import org.abaez.entities.FoodTruck;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.repository.ListCrudRepository;
+import org.springframework.stereotype.Repository;
+import java.util.List;
+import org.springframework.data.jpa.repository.Query;
+
+@Repository
+public interface FoodTruckRepository extends ListCrudRepository<FoodTruck, Integer>, JpaSpecificationExecutor<FoodTruck> {
+
```

Comment:
These methods are used to get the values which can be used in the filter parameters of the main query. An alternative to this would be to pull all the data locally and dedupe in the code. Doing it this way reduces the amount of data that has to be in flight and keeps the concerns properly separated.